addon/mixins/list-input.js

```diff
@@ -0,0 +1,71 @@
+// import _ from 'lodash'
+import Ember from 'ember'
+// import computed, {readOnly} from 'ember-computed-decorators'
+
+import * as utils from '../components/utils'
+
+export default Ember.Mixin.create({
+  init () {
+    this._super(...arguments)
+    this.set('options', [])
```

Comment:
Since options are used by the template should this be `Ember.A([])` instead. Otherwise I assume Ember is going to magically convert it to an Ember array anyway.